src/main.js

```diff
@@ -24,3 +24,44 @@ function printNumbersWithForEachLoop(arr) {
 printNumbersWithForLoop(10);
 printNumbersWithWhileLoop(10);
 printNumbersWithForEachLoop([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+
+//Objects
+var days= {
+    'monday' : 'Monday',
+    'tuesday' : 'Tuesday',
+    'wednesday' : 'Wednesday',
+    'thursday' : 'Thursday',
+    'friday' : 'Friday',
+    'saturday' : 'Saturday',
+    'sunday' : 'Sunday'
+}
+
+//Method 1
+let daysKeys= Object.keys(days);
+daysKeys.forEach((keys)=>console.log(keys));
+
+let daysValues= Object.values(days);
+daysValues.forEach((values)=> console.log(values))
+
+//Method 2--- some examples mentioned hasOwnProperty method with if condition?
+//Printing keys
+for(const key in days){
+    console.log(key);
+}
+
+//printing values
+for(const value in days){
+    console.log(days[value]);
+}
+
+//Printing length
+let daysLength= Object.keys(days).length;
+console.log(daysLength);
+
+//Replace key value
+days['monday'] = 'Somvar';
```

Comment:
Thing to experiment with -

`What if the variable 'days' was 'const' instead of 'var'?`
`Would we be able to re-assign value to one of its properties?`